src/shared/bus-unit-util.c

```diff
@@ -71,6 +71,11 @@ int bus_parse_unit_info(sd_bus_message *message, UnitInfo *u) {
                         &u->job_path);
 }
 
+static int warn_deprecated(const char *field, const char *eq) {
+        log_warning("DBus property %s is deprecated, ignoring assignment: %s=%s", field, field, eq);
```

Comment:
There's a caveat that this is not a "dbus property" here yet, it's a "transient setting", following the nomenclature established in #37665. But for those names that are added, the name is the same for the transient setting and the D-Bus property. So I guess that's fine.

But please change to "D-Bus". It's either "dbus" or "D-Bus".